tests/conftest.py

```diff
@@ -28,6 +28,8 @@
 import pytest
 import requests
 
+import cudf
+
```

Comment:
We really dont want this here if possible. It will slow everything down including just getting the tests name. Do a local import where its needed.